bootstrap.js

```diff
@@ -67,21 +63,39 @@ this.startup = function(data, reason) {
   // the actual controller
   urlInputControllers = urlInput.controllers;
   urlInputControllers.insertControllerAt(0, urlInputController);
+}
 
-  // Add the event listener to remove the CSS class when the animation ends
-  shareButton.addEventListener("animationend", shareButtonAnimationListener);
+this.install = function(data, reason) {};
+
+this.startup = function(data, reason) {
+  bWindow = Services.wm.getMostRecentWindow("navigator:browser");
+  bDocument = bWindow.document;
+
+  // See https://dxr.mozilla.org/mozilla-central/source/browser/base/content/browser-customization.js#11
```

Comment:
This should use a permalink (one that includes the commit hash), since it will likely change a t some point in the future. There is a link on the sidebar of DXR for that.